include/setup.php

```diff
@@ -317,11 +317,35 @@
 // Loads English localized strings by default (must go before config.php)
 require 'languages/english.php';
 
-// Get the user's personalised config (requires the locwebsvnhttp stuff above)
+// Support one WebSVN-installation hosting multiple different SVNParentPaths, distinguished  by their
+// location. Per location, the web server needs to set some environment variable providing the path
+// to the config to either exclusively or additionally include, implementing a simple layered config
+// this way. That allows e.g. changing paths to repos per location only and share everything else.
+//
+// The following implementation deals with multiple most likely problems in such an environment, like
+// HTTP-redirects influencing the name of the environment variable set and "preg_grep" indexing its
+// results depending on the input, so optionally changing between requests.
+//
+// https://stackoverflow.com/questions/3050444/when-setting-environment-variables-in-apache-rewriterule-directives-what-causes
+// https://bz.apache.org/bugzilla/show_bug.cgi?id=58739
+$confSuccess = 0;
+$envPathConf = preg_grep('/^(?:REDIRECT_)*WEBSVN_PATH_CONF$/', array_keys($_SERVER));
```

Comment:
Do you expect more than one `REDIRECT_` prefix?

Reply:
I had two in my case already for different reasons and the problem is we can't know how many redirects someone is using. So we simply shouldn't care at all.